lnd_multi-hop_htlc_remote_chain_claim_test.go

```diff
@@ -59,20 +77,33 @@ func testMultiHopHtlcRemoteChainClaim(net *lntest.NetworkHarness, t *harnessTest
 		t.Fatalf("unable to send payment: %v", err)
 	}
 
-	// We'll now wait until all 3 nodes have the HTLC as just sent fully
-	// locked in.
+	// At this point, all 3 nodes should now have an active channel with
+	// the created HTLC pending on all of them.
 	var predErr error
 	nodes := []*lntest.HarnessNode{net.Alice, net.Bob, carol}
 	err = lntest.WaitPredicate(func() bool {
-		predErr = assertActiveHtlcs(nodes, carolInvoice.RHash)
+		predErr = assertActiveHtlcs(nodes, payHash[:])
 		if predErr != nil {
 			return false
 		}
 
 		return true
 	}, time.Second*15)
 	if err != nil {
-		t.Fatalf("htlc mismatch: %v", err)
+		t.Fatalf("htlc mismatch: %v", predErr)
+	}
+
+	// Wait for carol to mark invoice as accepted. There is a small gap to
+	// bridge between adding the htlc to the channel and executing the exit
+	// hop logic.
+	for {
```

Comment:
this block might make a good helper function, since it's used 3+ times

Reply:
extracted